GomokuClient/packages/gomoku-core/src/api/schema.json

```diff
@@ -73,6 +73,24 @@
         "tags": ["Game"],
         "summary": "Get all games, which are available to join",
         "parameters": [
+          {
+            "name": "Limit",
+            "in": "query",
+            "style": "form",
+            "schema": {
+              "type": "integer",
+              "format": "int32"
+            }
+          },
+          {
+            "name": "Offset",
```

Comment:
I would prefer lowercase limit and offset for all our query parameters and to follow that convention. What you think?

Reply:
I noticed we also have on frontend White and Black also changed to uppercase. I'm absolutely positive to have consistency and follow that kind of format. But ussualy lowercase query params and payloads are preffered.
https://github.com/microsoft/api-guidelines?tab=readme-ov-file